interface/chat/.gitignore

```diff
@@ -0,0 +1,30 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
`.gitignore` goes in the top level directory.

Reply:
Also, since this is going in a subdirectory, some of these paths may change for you?